cmd/terramate/cli/run.go

```diff
@@ -327,6 +327,11 @@ func (c *cli) runAll(
 				printScriptCommand(c.stderr, run.Stack, task)
 			}
 
+			if !opts.Quiet && !opts.ScriptRun {
+				printer.Stderr.Println(printPrefix + " Entering stack in " + run.Stack.String())
+				printer.Stderr.Println(printPrefix + " Executing command " + strconv.Quote(cmdStr))
+			}
+
```

Comment:
Why this was moved up? now it prints `Executing command ...` but it's not about to run but still need to execute cloud related stuff. So maybe the user can see things like:
```
Executing command ...
<cloud related print or cloud error>
<command is executed>
```